src/main/resources/static/js/MainWindow.js

```diff
@@ -774,558 +748,538 @@ var initial = 1000;
+        $.notify(
+          {
+            message:
+              "Please specify the date format (no empty strings allowed)."
+          },
+          {
+            allow_dismiss: true,
+            type: "info",
+            placement: { from: "top", align: "left" },
+            animate: {
+              enter: "animated fadeInDown",
+              exit: "animated fadeOutUp"
+            }
+          }
+        );
+        stop = true;
+        return false;
+      }
+      obj["string"] = currentInput;
+      currentInput = $td
+        .eq(0)
+        .find("input")
+        .val();
+      if (currentInput === null || currentInput === "") {
+        $.notify(
+          {
+            message:
+              "Please specify the column where the date can be found (no empty field allowed) before importing data."
+          },
+          {
+            allow_dismiss: true,
+            type: "info",
+            placement: { from: "top", align: "left" },
+            animate: {
+              enter: "animated fadeInDown",
+              exit: "animated fadeOutUp"
+            }
+          }
+        );
+        stop = true;
+        return false;
+      }
+      parsed = parseInt(currentInput, 10);
+      if (!(currentInput == parsed) || parsed < 0) {
+        $.notify(
+          {
+            message:
+              "Please specify the column where the date can be found (must be a non-negative number) before importing data."
+          },
+          {
+            allow_dismiss: true,
+            type: "info",
+            placement: { from: "top", align: "left" },
+            animate: {
+              enter: "animated fadeInDown",
+              exit: "animated fadeOutUp"
+            }
+          }
+        );
+        stop = true;
+        return false;
+      }
+      obj["column"] = parsed;
+      date.push(obj);
+    });
+  if (stop === true) {
+    return false;
+  }
+  var streams = [];
+  $("#datastreams")
+    .find(".datastream")
+    .each(function() {
+      var obj = {},
+        obs = [];
+      currentInput = $(this)
+        .find("select option:selected")
+        .val();
+      if (currentInput == null || currentInput == "") {
+        $.notify(
+          { message: "Please leave no datastream empty." },
+          {
+            allow_dismiss: true,
+            type: "info",
+            placement: { from: "top", align: "left" },
+            animate: {
+              enter: "animated fadeInDown",
+              exit: "animated fadeOutUp"
+            }
+          }
+        );
+        stop = true;
+        return false;
+      }
+      obj["dsID"] = JSON.parse(
+        $(this)
+          .find("select option:selected")
+          .attr("data-value")
+      ).frostId;
+      $(this)
+        .find("tbody tr")
+        .each(function() {
+          currentInput = $(this)
+            .find("td:eq(1) input")
+            .val();
+          if (currentInput == null || currentInput == "") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4898788)